src/libappimage/utils/Logger.cpp

```diff
@@ -0,0 +1,49 @@
+// libraries
```

Comment:
So, instead of re-using existing code, you thought it was wise to create yet another system? We had agreed on being able to hook your own logging system by using factories. This doesn't at all provide such functionality, and is basically copying ldLog. It's not providing any flexibility.

Reply:
I aware of it, but still can't find a way of making it all fit together in a nice way. That's why I opted for a simple logger. We already have an issue for logs so we can discuss it later.